modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -16,30 +16,20 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import {Editor} from './Editor';
+import DEFAULT_BALLOON_EDITOR_CONFIG from './config/DefaultBalloonEditorConfiguration';
 
 import '../css/main.scss';
 
-const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
-	const defaultExtraPlugins = 'ballooneditor,videoembed';
-
-	const extraPlugins = config.extraPlugins ? `${config.extraPlugins},` : '';
-
-	const basicToolbars = {
-		toolbarImage:
-			'ImageAlignLeft,ImageAlignCenter,ImageAlignRight,LinkAddOrEdit,AltImg',
-		toolbarTable: 'TableHeaders,TableRow,TableColumn,TableCell,TableDelete',
-		toolbarText:
-			'Styles,Bold,Italic,Underline,BulletedList,NumberedList,TextLink,' +
-			'JustifyLeft,JustifyCenter,JustifyRight,LineHeight,RemoveFormat',
-		toolbarVideo: 'VideoAlignLeft,VideoAlignCenter,VideoAlignRight',
-	};
-
+const BalloonEditor = ({
+	baseConfig = DEFAULT_BALLOON_EDITOR_CONFIG,
```

Comment:
Does `baseConfig` need to be a prop? :thinking: `config` should be enough to allow setting config overrides.

Reply:
![image](https://user-images.githubusercontent.com/7663513/129405532-9139104e-de0e-4f4e-9529-a8a53cca0b4e.png)

Reply:
![image](https://user-images.githubusercontent.com/7663513/129405560-082d1976-cb02-4124-a096-0ff9a4804a2d.png)

Reply:
> Does baseConfig need to be a prop?

Nope! Removing!

Reply:
force-pushing!

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/1352/commits/19d98168a13fda64e4e29e25c43ebe3bdd685c89